notebooks/data_downloader.py

```diff
@@ -0,0 +1,12 @@
+import os
+import git
+from git.repo.base import Repo
+cwd = os.getcwd()
+## we are creating a folder with the name 'data' inside your current working directory. 
+path_to_save = cwd+'/data'
```

Comment:
There is already a `data` folder in this repo. Running this script in any directory besides the topmost will create an additional `data` directory.  `path_to_save` should be `/data/<kind of data>` where kind of data is raw, interim, processed or raw.

Reply:
I created a script by considering user don't have the entire repo structure on his system. 
If users have an entire repo, it makes sense to add the suggested changes i.e. a script to `src/data` and download the data in the folder `data/raw`

Should I make these changes?

Reply:
Yes, I think you should make the change. You should assume the user has this script because they have cloned the repo its in.